pandera/schemas.py

```diff
@@ -6,22 +6,22 @@
 import warnings
 from functools import wraps
 from pathlib import Path
-from typing import Any, Callable, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
 import pandas as pd
 from packaging import version
 
-from . import constants, dtypes, errors
-from .checks import Check
-from .dtypes import PandasDtype, PandasExtensionType
-from .error_formatters import (
+from pandera import constants, dtypes, errors
```

Comment:
these should be reverted to relative imports

Reply:
sure thing -- this is a remnant of my environment set up issues. will revert on next commit.